src/cdk-experimental/clipboard/clipboard.ts

```diff
@@ -70,11 +70,14 @@ export class PendingCopy {
 
   constructor(text: string, private readonly _document: Document) {
     const textarea = this._textarea = this._document.createElement('textarea');
+    const styles = textarea.style;
 
-    // Hide the element for display and accessibility.
-    textarea.setAttribute('style', 'opacity: 0;');
+    // Hide the element for display and accessibility. Set an
+    // absolute position so the page layout isn't affected.
+    styles.opacity = '0';
+    styles.position = 'absolute';
+    styles.left = styles.top = '-999em';
```

Comment:
Just out of curiosity: why `em`?

Reply:
No particular reason. I just wanted to make sure that it's off-screen so I put in something that is guaranteed to be large.